db/import_mb_data.py

```diff
@@ -0,0 +1,1512 @@
+
+        # END AREA
+        try:
+            load_end_area(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # AREA TYPE
+        try:
+            load_area_type(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # BEGIN AREA TYPE
+        try:
+            load_begin_area_type(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # END AREA TYPE
+        try:
+            load_end_area_type(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # ARTIST GID REDIRECT
+        try:
+            load_artist_gid_redirect(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # GENDER
+        try:
+            load_gender(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # LANGUAGE
+        try:
+            load_language(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # MEDIUM FORMAT
+        try:
+            load_medium_format(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # RECORDING GID REDIRECT
+        try:
+            load_recording_gid_redirect(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # release_group gid redirect
+        try:
+            load_release_group_gid_redirect(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # release_group
+        try:
+            load_release_group(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # release_group_primary_type
+        try:
+            load_release_group_primary_type(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # release_packaging
+        try:
+            load_release_packaging(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # release_status
+        try:
+            load_release_status(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+        # script
+        try:
+            load_script(connection, gids_in_AB)
+        except ValueError:
+            print("No Data found for the recordings")
+
+    insert_MB_data_AB()
+    print("--------------------------------DONE-----------------------------------")
+
+
+def start_import():
+    with db.engine.begin() as connection:
+        lowlevel_query = text("""SELECT gid from lowlevel""")
```

Comment:
Could we get these gids in batches too?

``` 
SELECT gid  FROM lowlevel ORDER BY id OFFSET 0 LIMIT 10000
SELECT gid  FROM lowlevel ORDER BY id OFFSET 0 LIMIT 10000
```
and so on.

Reply:
Added.